removed stand alone battery sensor & updated icon

CBDesignS opened the pull request:
- Removed stand alone battery sensor and added to sensors.py  so it can be imported into vacuum.py along with any other sensors that might be found and added in the future. i.e water level etc.
- updated icon from eufy mqtt to eufy clean to align with addon name.
- fixed .lower bug that has been stable on my docker test HA and my current working HA server. (to fix this vacuum.py, sharedconnect.py & mqttconnect.py all had to fixed to be compatable
- tried to add more debugging code and notiations for future work.

jeppesens commented:
@CBDesignS most of this code is now released here https://github.com/jeppesens/eufy-clean/releases/tag/v1.1.0

CBDesignS commented:
I have moved sensor.py into __init__.py away from vacuum.py so it is where it should be so HA can use it and I can ask Alexa what the robovac battery level is and it replies with the current charge level, hence the reason to get other sensors included correctly,

I hope to get water level done so I can ask the water level and have an automation that will send a notification to my phone and Alexa telling me there is less than 20% water remaining etc etc. I pulled your code back down again so I have a clean start to dive into the sensors and see if I can get to be where I want to be with it.

I re-done the icon with an image of an x10 robovac on it and changed the wording to eufy clean instead of eufy mqtt that I had the first time.
I have been informed that the icon will not show up because you are a fork of the main repo and only the main repo can have the icon and be submitted to hacs bla bla etc etc. They told me the way round it is to download your repo so you have it safe. Remove your forked repo to break the link to the main repo. Build your new eufy clean repo and upload your code, that way you become the main repo and have control over the icon files and hacs, you will loose the ability to pull any code the main repo posts but I don`t know if its still under development there..

Chris


________________________________
From: Tobias Jeppesen ***@***.***>
Sent: Tuesday, May 27, 2025 21:03
To: jeppesens/eufy-clean ***@***.***>
Cc: CBDesignS ***@***.***>; Mention ***@***.***>
Subject: Re: [jeppesens/eufy-clean] removed stand alone battery sensor & updated icon (PR #38)

[https://avatars.githubusercontent.com/u/29945975?s=20&v=4]jeppesens left a comment (jeppesens/eufy-clean#38)<https://github.com/jeppesens/eufy-clean/pull/38#issuecomment-2913868002>

@CBDesignS<https://github.com/CBDesignS> most of this code is now released here https://github.com/jeppesens/eufy-clean/releases/tag/v1.1.0

—
Reply to this email directly, view it on GitHub<https://github.com/jeppesens/eufy-clean/pull/38#issuecomment-2913868002>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANBO4KRX5EM7YQOEWA33EMT3ATAHRAVCNFSM6AAAAAB6AX6OGSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMJTHA3DQMBQGI>.
You are receiving this because you were mentioned.
